en/index.md

```diff
@@ -32,11 +32,14 @@ Welcome to the Cocos Creator User Manual! Cocos Creator is a free, cross-platfor
 ## Example Projects
 
 - [Example Collections](https://github.com/cocos-creator/example-cases): You can also get access to it by creating a new project using that template. It's a case by case introduction of most Cocos Creator components and features.
-- [Your first Cocos Creator game: Star Catcher](https://github.com/cocos-creator/tutorial-first-game)，please read [Quick Start Tutorial](getting-started/quick-start.md) to build it step by step!
+- [Your first Cocos Creator game: Star Catcher](https://github.com/cocos-creator/tutorial-first-game)，please read [Quick Start Tutorial](getting-started- Starting with v1.9, Cocos Creator support posted to [Facebook Instant Games](publish/publish-fb-instant-games.md) and [QQ Play](publish/publish-qqplay.md).
+- Starting with v1.8，Cocos Creator support posted to Wechat Mini Games，please refer to [Wechat Mini Games](publish/publish-wechatgame.md).
+- Cocos Creator also supports exporting scenes and UI to Cocos2d-x, see [C++/Lua Support](advanced-topics/cpp-lua.md).
+/quick-start.md) to build it step by step!
```

Comment:
这部分内容是 Example Projects，新增的这几个跳转链接都不用放在这里，是贴错了吗？
![image](https://user-images.githubusercontent.com/19702899/49565667-e93cfe80-f962-11e8-888e-aace3e9154e5.png)

Reply:
I don't know why this is happening. I am going to switch to VS Code today. I apologize. I am finding random text in files and it doesn't make sense to my why. I've never had this problem in any other documentation but now I am running Atom on Linux instead of macOS.